### PR TITLE
Updates to Isochrone API

### PR DESCRIPTION
**Pull request by cafesilencio**
Added updates to Isochrone API to support new features like driving-traffic profile, contours_meters,  road/route exclusions and departure date/time.

### CODE REVIEW

File: services-isochrone/src/main/java/com/mapbox/api/isochrone/MapboxIsochrone.java [hunk 1]
```diff
@@ -217,6 +239,23 @@ public Builder addContoursMinutes(@NonNull @IntRange(from = 0, to = 60)
...
+     * The distances, in meters, to use for each isochrone contour. You
+     * must be in increasing order. The maximum distance that can be specified
+     * is 100000 meters (100km).
+     *
+     * @param listOfMeterValues an integer list with at least one number
+     *                           for the meters which represent each contour
+     * @return this builder for chaining options together
+     * @since 7.3.0
+     */
+    public Builder addContoursMeters(@NonNull @IntRange(from = 0, to = 100000)
```
Comment (LukasPaczos) [inline services-isochrone/src/main/java/com/mapbox/api/isochrone/MapboxIsochrone.java:253]: Let's not add the integer range, this range can be dynamically changed on the backend.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### main
 
+### v7.3.0 - September 26, 2024
```
Comment (LukasPaczos) [inline CHANGELOG.md:7]:
```suggestion
```

This isn't yet released :)

File: services-isochrone/src/main/java/com/mapbox/api/isochrone/MapboxIsochrone.java [hunk 3]
```diff
@@ -217,6 +239,23 @@ public Builder addContoursMinutes(@NonNull @IntRange(from = 0, to = 60)
...
+                                     Integer... listOfMeterValues) {
```
Comment (LukasPaczos) [inline services-isochrone/src/main/java/com/mapbox/api/isochrone/MapboxIsochrone.java:254]:
```suggestion
    public Builder addContoursMeters(@NonNull Integer... listOfMeterValues) {
```
Let's not add the integer range, this range can be dynamically changed on the backend.

File: services-isochrone/src/main/java/com/mapbox/api/isochrone/MapboxIsochrone.java [hunk 4]
```diff
@@ -217,6 +239,23 @@ public Builder addContoursMinutes(@NonNull @IntRange(from = 0, to = 60)
       return this;
     }
 
+    /**
+     * An integer list of meter values to use for each isochrone contour.
+     * The distances, in meters, to use for each isochrone contour. You
+     * must be in increasing order. The maximum distance that can be specified
+     * is 100000 meters (100km).
```
Comment (LukasPaczos) [inline services-isochrone/src/main/java/com/mapbox/api/isochrone/MapboxIsochrone.java:246]:
```suggestion
     * must be in increasing order. The default maximum distance that can be specified
     * is 100000 meters (100km), if you need a bigger range contact support.
```

File: services-isochrone/src/main/java/com/mapbox/api/isochrone/MapboxIsochrone.java [hunk 5]
```diff
@@ -226,12 +265,25 @@ public Builder addContoursMinutes(@NonNull @IntRange(from = 0, to = 60)
...
     // Required for matching with MapboxIsochrone addContoursMinutes() method.
     @SuppressWarnings("WeakerAccess")
     abstract Builder contoursMinutes(@NonNull String stringListOfMinuteValues);
 
+    /**
+     * A single String which is a comma-separated list of values(s) in meters
+     * to use for each isochrone contour. The distances, in meters, to use for each
+     * isochrone contour. You can specify up to four contours. Distances must be in
+     * increasing order. The maximum distance that can be specified
+     * is 100000 meters (100km).
```
Comment (LukasPaczos) [inline services-isochrone/src/main/java/com/mapbox/api/isochrone/MapboxIsochrone.java:279]:
```suggestion
     * increasing order. The maximum distance that can be specified
     * is 100000 meters (100km), if you need a bigger range contact support.
```

File: CHANGELOG.md (hunk 2 above)
Comment (cafesilencio) [reply CHANGELOG.md:7]: Looking at the release page of the repo the current release is 7.2.0 so won't the next release be 7.3.0? What should this be?

Comment (Zayankovsky) [reply CHANGELOG.md:7]: I think we are supposed to put changelog entries into the `main` section. Then the release train will create a section with a proper version name and date and move pending entires there

Comment (cafesilencio) [reply CHANGELOG.md]: I changed it. Is it correct? I moved an entry from a PR I merged earlier in the week.

**Review by Zayankovsky — APPROVED** [review]

**Pull request merged**